administration/console.md

```diff
@@ -20,7 +20,13 @@ The method to access the console depends on how openHAB was started.
 * When started in interactive mode using the provided command line scripts, openHAB naturally transitions directly to the console prompt.
 * When started as a service (e.g. when installed from our package repository), openHAB is running as a background process.
 
-In both cases, the console can be reached via secure shell connection ([SSH](https://en.wikipedia.org/wiki/Secure_Shell)).
+The simplest way of accessing the console is by running the `client` (or `client.bat` for windows) script located in `$OPENHAB_RUNTIME/bin/`. Linux package based installations can also use the command `openhab-cli console`.
```

Comment:
Consider, "In this case, the simplest way of accessing...", and if there is a blank line between the sentence, "When started as a service..." and the line we are talking about, consider deleting it so this becomes one paragraph.